apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Logging/LoggingTest.cs

```diff
@@ -387,21 +643,23 @@ public async Task Logging_ServiceContext_VersionOnly_Exception()
     /// </summary>
     public abstract class LoggerTestApplication
     {
-        protected string ProjectId;
+        protected readonly string _projectId = TestEnvironment.GetTestProjectId();
+        protected readonly double _traceQps;
 
         public LoggerTestApplication()
-        {
-            ProjectId = TestEnvironment.GetTestProjectId();
-        }
+            // Don't actually trace anything unless there's a header that forces tracing.
+            : this(0.00000001)
```

Comment:
```suggestion
            : this(double.Epsilon)
```

(I'm assuming this can't actually be 0? Perhaps we should fix that?)

Reply:
I've had to revert this, using Epsilon overflows a TimeSpan somewhere else. Will look into accepting small enough values, including zero, in another PR.